src/fuzz_introspector/analyses/sinks_analyser.py

```diff
@@ -0,0 +1,394 @@
+            1,
+            toc_list
+        )
+
+        # Table with all function calls for each files
+        html_string += "<div class=\"collapsible\">"
+        html_string += (
+            "<p>"
+            "This section shows a chosen list of functions / methods "
+            "calls and their relative coverage information. By static "
+            "analysis of the target project code, all of these function "
+            "call and their caller information, including the source file "
+            "or class and line number that initiate the call are captured. "
+            "The caller source code file or class and the line number are "
+            "shown in column 2 while column 1 is the function name of that "
+            "selected functions or methods call. Each occurrent of the target "
+            "function call will occuply a separate row. Column 3 of each row "
+            "indicate if the target function calls is statically unreachable."
+            "Column 4 lists all fuzzers (or no fuzzers at all) that have "
+            "covered that particular system call in  dynamic fuzzing. Those "
+            "functions with low to  no reachability and dynamic hit count indicate "
+            "missed fuzzing logic to fuzz and track for possible code injection sinks."
+            "</p>"
+        )
+
+        html_string += html_helpers.html_add_header_with_link(
+            "Function in each files in report",
+            2,
+            toc_list
+        )
+
+        # Third party function calls table
+        tables.append(f"myTable{len(tables)}")
+        html_string += html_helpers.html_create_table_head(
+            tables[-1],
+            [
+                ("Target sink", ""),
+                ("Callsite location",
+                 "Source file, line number and parent function of this function call. "
+                 "Based on static analysis."),
+                ("Reached by fuzzer",
+                 "Is this code reachable by any functions? "
+                 "Based on static analysis."),
+                ("Covered by Fuzzers",
+                 "The specific list of fuzzers that cover this function call. "
+                 "Based on dynamic analysis.")
+            ]
+        )
+
+        for fd in self.filter_function_list(function_list, profiles[0].target_lang):
+            # Loop through the list of calledlocation for this function
+            for called_location in function_callsite_dict[fd.function_name]:
+                # Determine if this called location is covered by any fuzzers
+                fuzzer_hit = False
+                coverage = proj_profile.runtime_coverage
+                for parent_func in fd.incoming_references:
+                    try:
+                        lineno = int(called_location.split(":")[1])
+                    except ValueError:
+                        continue
+                    if coverage.is_func_lineno_hit(parent_func, lineno):
```

Comment:
Am not sure this will work for all languages and whether we should have more insights about the target language of the coverage profile, or, alternatively do some refactoring. Am happy with addressing this at a later stage.